src/main/java/org/apache/accumulo/testing/continuous/ContinuousIngest.java

```diff
@@ -86,51 +82,49 @@ private static int getFlushEntries(Properties props) {
     return Integer.parseInt(props.getProperty(TestProps.CI_INGEST_FLUSH_ENTRIES, "1000000"));
   }
 
-  private static void pauseCheck(Properties props, Random rand) throws InterruptedException {
-    if (pauseEnabled(props)) {
+  private static void pauseCheck(Random rand) throws InterruptedException {
+    if (pauseEnabled) {
       long elapsedNano = System.nanoTime() - lastPauseNs;
       if (elapsedNano > (TimeUnit.SECONDS.toNanos(pauseWaitSec))) {
-        long pauseDurationSec = getPause(props, rand, CI_INGEST_PAUSE_DURATION_MIN,
-            CI_INGEST_PAUSE_DURATION_MAX);
-        log.info("PAUSING for " + pauseDurationSec + "s");
+        long pauseDurationSec = getPause(rand);
+        log.info("PAUSING for {}s", pauseDurationSec);
         Thread.sleep(TimeUnit.SECONDS.toMillis(pauseDurationSec));
         lastPauseNs = System.nanoTime();
-        pauseWaitSec = getPause(props, rand, CI_INGEST_PAUSE_WAIT_MIN, CI_INGEST_PAUSE_WAIT_MAX);
-        log.info("INGESTING for " + pauseWaitSec + "s");
+        pauseWaitSec = getPause(rand);
+        log.info("INGESTING for {}s", pauseWaitSec);
       }
     }
   }
 
   public static void main(String[] args) throws Exception {
 
-    try (ContinuousEnv env = new ContinuousEnv(args)) {
+    try (ContinuousEnv env = new ContinuousEnv(args);
+        AccumuloClient client = env.getAccumuloClient()) {
 
-      visibilities = parseVisibilities(env.getTestProperty(TestProps.CI_INGEST_VISIBILITIES));
-
-      long rowMin = env.getRowMin();
-      long rowMax = env.getRowMax();
+      final long rowMin = env.getRowMin();
+      final long rowMax = env.getRowMax();
       if (rowMin < 0 || rowMax < 0 || rowMax <= rowMin) {
         throw new IllegalArgumentException("bad min and max");
       }
 
-      AccumuloClient client = env.getAccumuloClient();
       String tableName = env.getAccumuloTableName();
       if (!client.tableOperations().exists(tableName)) {
         throw new TableNotFoundException(null, tableName,
             "Consult the README and create the table before starting ingest.");
       }
 
-      BatchWriter bw = client.createBatchWriter(tableName);
-
-      Random r = new Random();
+      Random rand = new Random();
 
       byte[] ingestInstanceId = UUID.randomUUID().toString().getBytes(UTF_8);
+      log.info("Ingest instance ID: {} current time: {}ms", new String(ingestInstanceId, UTF_8),
```

Comment:
I like the new log messages but I am not quite sure the usefulness of this one. Especially about current time. It is possible we could shift some of the log message to debug level but considering these only get displayed once per command run, it isn't necessary.

Reply:
Yea I agree, I'm not sure that I see the usefulness of this log, I just left it in since it was already there and made it more descriptive.

Reply:
Not sure how useful the time is, but the UUID is very useful to know.  When data is lost there will be UUID in the accumulo table pointing to the missing data.  Can use that to fine the ingest logs if those logs contain the UUID.

Reply:
Gotcha, thanks. I think I'll leave this log as it appears above just in case the timestamp could be useful.